Sources/View/UITableView+Reusable.swift

```diff
@@ -40,25 +40,61 @@ public extension UITableView {
 
    - parameter indexPath: The index path specifying the location of the cell.
    - parameter cellType: The cell class to dequeue
+   - parameter autoRegister: Set to true in order to automatically register the cell class/nib if not registered
 
    - returns: A `Reusable`, `UITableViewCell` instance
 
    - note: The `cellType` parameter can generally be omitted and infered by the return type,
    except when your type is in a variable and cannot be determined at compile time.
    - seealso: `dequeueReusableCell(withIdentifier:,for:)`
    */
-  final func dequeueReusableCell<T: UITableViewCell>(for indexPath: IndexPath, cellType: T.Type = T.self) -> T
+    final func dequeueReusableCell<T: UITableViewCell>(
```

Comment:
Please use 2-spaces indentation for consistency's stake with the rest of the code (I'm not strongly an advocate for 2 vs 4 spaces, but since we use 2 everywhere else, better keep it consistent)